core/arch/arm/plat-ti/platform_config.h

```diff
@@ -45,8 +45,10 @@
 
 #if defined(PLATFORM_FLAVOR_dra7xx)
 #define CONSOLE_UART_BASE       UART1_BASE
-#elif
+#elif defined(PLATFORM_FLAVOR_am57xx)
 #define CONSOLE_UART_BASE       UART3_BASE
+#else
+#error "Unknown platform flavor"
```

Comment:
We can only enter this section if ```defined(PLATFORM_FLAVOR_dra7xx) || defined(PLATFORM_FLAVOR_am57xx)``` so this may not be needed. A better fix for this patch would be to instead change the ```#elif``` to ```#else``` as this is what I intended. I have no strong preference either way though.